app/src/main/java/org/oppia/app/player/exploration/ExplorationContentViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.app.player.exploration
+
+/** [ViewModel] for [StatePlayerRecyclerViewAssembler] items. */
+class ExplorationContentViewModel(hasBlueBackground: Boolean, isCenterAligned: Boolean) {
+  var hasBlueBackground: Boolean = false
+  var isCenterAligned: Boolean = false
```

Comment:
Makes this values as observable fields and control those values instead of using `init`

```
  val hasBlueBackground = ObservableField<Boolean>(false)
  val isCenterAligned = ObservableField<Boolean>(false)
```